app/src/main/java/com/neudesic/myapplication/domain/repository/DadJokeRepository.kt

```diff
@@ -0,0 +1,8 @@
+package com.neudesic.myapplication.domain.repository
+
+import com.neudesic.myapplication.domain.models.DadJoke
+import retrofit2.Response
+
+interface DadJokeRepository {
+    suspend fun getJoke(): Response<DadJoke>?
```

Comment:
I'm thinking the `Response` should not be null. Instead the `DadJoke`

Reply:
I updated it but its not optional on either.
I realized that if GSON fails to convert, it will just be an empty Model of DadJoke.

I don't think its possible to get a null return type from retrofit & GSON (?)

Reply:
That sounds right